uncertainties/core.py

```diff
@@ -65,139 +66,151 @@
 
 try:
     import numpy
+
+    __all__ += [
+        "correlated_values",
+        "correlated_values_norm",
+        "correlation_matrix",
+    ]
```

Comment:
Yeah, in a separate PR we should get rid of that `from .core import *` and just explicitly define the public API there. It will help with various automated code inspections.